### ISSUE TITLE
Missing ';' in holberton.h header file

### ISSUE THREAD

**Issue by narnat** [kai-dg/hipposcraper#44]
It misses ';' on holberton.h header file

**Comment by kai-dg** [comment]
I need the project link and exact item thats missing.

**Comment by kai-dg** [comment]
> It misses ';' on holberton.h header file

I need the project link and exact item thats missing.

**Comment by narnat** [comment]
here's the link to the project https://intranet.hbtn.io/projects/222

**Comment by narnat** [comment]
```
#ifndef HOLBERTON_H
#define HOLBERTON_H

#include <stdio.h>
#include <stdlib.h>

int _putchar(char c);
char *create_array(unsigned int size, char c)
char *_strdup(char *str)
char *str_concat(char *s1, char *s2)
int **alloc_grid(int width, int height)
void free_grid(int **grid, int height)
char *argstostr(int ac, char **av)
char **strtow(char *str)

#endif /* HOLBERTON_H */
```

there is only one ;

**Comment by kai-dg** [comment]
thank you, fixed on #45
